### PR TITLE
Localize policy as code #1653

### PR DESCRIPTION
**Pull request by matteobisi**
### Describe your changes
I've added the IT localization for "policy as a code"

### Related issue number or link (ex: `resolves #issue-number`)

#1653 
### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/it/policy-as-code.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Policy as code è la pratica che consente di memorizzare la definizione delle policy in uno o più file in formato leggibile e processabile da macchine. 
+Questa metodologia sostituisce il modello tradizionale in cui le policy sono documentate in formato leggibile da umani in documenti esterni al sistema.
+
+## Quali problematiche affronta
+
+La costruzione di applicazioni e infrastrutture è spesso vincolata da molte policy che un'organizzazione definisce, 
+ad esempio policy di sicurezza che vietano di memorizzare segreti nel codice sorgente, di eseguire un container con autorizzazioni di superutente 
+o di memorizzare alcuni dati al di fuori di una specifica regione geografica. 
+Il processo di verifica manuale della loro appliaczione su applicazioni o infrastruttura da parte di sviluppatori e revisori
```
Comment (annalisag-spark) [inline content/it/policy-as-code.md:19]:
```suggestion
 Per gli sviluppatori e i revisori è molto laborioso e soggetto a errori verificare manualmente le applicazioni e le infrastrutture rispetto alle policy documentate.
```

File: content/it/policy-as-code.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+Policy as code è la pratica che consente di memorizzare la definizione delle policy in uno o più file in formato leggibile e processabile da macchine. 
+Questa metodologia sostituisce il modello tradizionale in cui le policy sono documentate in formato leggibile da umani in documenti esterni al sistema.
+
+## Quali problematiche affronta
+
+La costruzione di applicazioni e infrastrutture è spesso vincolata da molte policy che un'organizzazione definisce, 
+ad esempio policy di sicurezza che vietano di memorizzare segreti nel codice sorgente, di eseguire un container con autorizzazioni di superutente 
+o di memorizzare alcuni dati al di fuori di una specifica regione geografica. 
+Il processo di verifica manuale della loro appliaczione su applicazioni o infrastruttura da parte di sviluppatori e revisori 
+è complesso e soggetto a errori.
```
Comment (annalisag-spark) [inline content/it/policy-as-code.md:20]:
```suggestion

```

File: content/it/policy-as-code.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+La costruzione di applicazioni e infrastrutture è spesso vincolata da molte policy che un'organizzazione definisce, 
+ad esempio policy di sicurezza che vietano di memorizzare segreti nel codice sorgente, di eseguire un container con autorizzazioni di superutente 
+o di memorizzare alcuni dati al di fuori di una specifica regione geografica. 
+Il processo di verifica manuale della loro appliaczione su applicazioni o infrastruttura da parte di sviluppatori e revisori 
+è complesso e soggetto a errori. 
+I processi manuali non possono soddisfare i requisiti di reattività e scalabilità delle applicazioni native del cloud.
+
+## In che modo aiuta
+
+Utilizzare il policy as a code consente la ripetibilità ed automazione, riducendo gli errori rispetto alla loro applicazione manuale.
```
Comment (annalisag-spark) [inline content/it/policy-as-code.md:25]:
```suggestion
Utilizzare Policy as Code la ripetibilità e riduce gli errori, a differenza di quanto avviene manualmente.
```

**Review by annalisag-spark — CHANGES_REQUESTED** [review]
🙂

File: content/it/policy-as-code.md (hunk 3 above)
Comment (annalisag-spark) [reply content/it/policy-as-code.md:24]: togliere quell'_a_ che precede Code

**Review by annalisag-spark — CHANGES_REQUESTED** [review]

File: content/it/policy-as-code.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+Policy as code è la pratica che consente di memorizzare la definizione delle policy in uno o più file in formato leggibile e processabile da macchine. 
+Questa metodologia sostituisce il modello tradizionale in cui le policy sono documentate in formato leggibile da umani in documenti esterni al sistema.
+
+## Quali problematiche affronta
+
+La costruzione di applicazioni e infrastrutture è spesso vincolata da molte policy che un'organizzazione definisce, 
+ad esempio policy di sicurezza che vietano di memorizzare segreti nel codice sorgente, di eseguire un container con autorizzazioni di superutente 
+o di memorizzare alcuni dati al di fuori di una specifica regione geografica. 
+ Per gli sviluppatori e i revisori è molto laborioso e soggetto a errori verificare manualmente le applicazioni e le infrastrutture rispetto alle policy documentate.
```
Comment (ugho16) [inline content/it/policy-as-code.md:19]:
```suggestion
Per gli sviluppatori e i revisori è molto laborioso e soggetto a errori verificare manualmente le applicazioni e le infrastrutture rispetto alle policy documentate.
```

File: content/it/policy-as-code.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+I processi manuali non possono soddisfare i requisiti di reattività e scalabilità delle applicazioni native del cloud.
+
+## In che modo aiuta
+
+Utilizzare Policy as Code la ripetibilità e riduce gli errori, a differenza di quanto avviene manualmente.
```
Comment (ugho16) [inline content/it/policy-as-code.md:24]:
```suggestion
Utilizzare Policy as Code consente la ripetibilità e riduce gli errori, a differenza di quanto avviene manualmente.
```

File: content/it/policy-as-code.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+---
+
+## Cos'è
+
+Policy as code è la pratica che consente di memorizzare la definizione delle policy in uno o più file in formato leggibile e processabile da macchine. 
+Questa metodologia sostituisce il modello tradizionale in cui le policy sono documentate in formato leggibile da umani in documenti esterni al sistema.
+
+## Quali problematiche affronta
+
+La costruzione di applicazioni e infrastrutture è spesso vincolata da molte policy che un'organizzazione definisce,
```
Comment (ugho16) [inline content/it/policy-as-code.md:16]:
```suggestion
La costruzione di applicazioni e infrastrutture è spesso vincolata a molte policy che un'organizzazione definisce, 
```

**Review by ugho16 — CHANGES_REQUESTED** [review]

**Review by ugho16 — APPROVED** [review]

**Review by annalisag-spark — APPROVED** [review]

**Pull request merged**